source/operator.rst

```diff
@@ -9,7 +9,7 @@ The operator is configured via the spec section of the
 The metadata part of this file contains the following keys:
 
 * ``name`` (``my-cluster-name`` by default) sets the name of your Percona Server
-  for MongoDB Cluster,
+  for MongoDB Cluster; it should include only `URL-compatible characters <https://datatracker.ietf.org/doc/html/rfc3986#section-2.3>`_ and should not exceed 22 characters;
```

Comment:
This is not fully correct. Try setting the name like 123cluster, Operator will throw you the error:
```
{"level":"error","ts":1622535167.1972747,"logger":"controller","msg":"Reconciler error","controller":"perconaxtradbcluster-controller","name":"123cluster","namespace":"default","error":"create PXC ServiceUnready: create: Service \"123cluster-pxc-unready\" is invalid: metadata.name: Invalid value: \"123cluster-pxc-unready\": a DNS-1035 label must consist of lower case alphanumeric characters or '-', start with an alphabetic character, and end with an alphanumeric character (e.g. 'my-name',  or 'abc-123', regex used for validation is '[a-z]([-a-z0-9]*[a-z0-9])?')","
```

Reply:
The PR here also has the same flaw: https://github.com/percona/percona-xtradb-cluster-operator/pull/887